gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskUpdateNotifier.java

```diff
@@ -23,14 +27,25 @@ public class TaskUpdateNotifier {
     private final StreamBridge streamBridge;
     private final TaskDtoBuilder taskDtoBuilder;
 
-    public TaskUpdateNotifier(StreamBridge streamBridge, TaskDtoBuilder taskDtoBuilder) {
+    @Value("${stomp.notify}")
+    private String notify;
+
+    private final SimpMessagingTemplate broker;
```

Comment:
Maybe rename broker to something more explicit, maybe stompBridge ? If you have any better idea, be pleased :)

Reply:
renamed to stompBridge